pkg/asset/installconfig/openstack/validation/platform.go

```diff
@@ -127,3 +132,28 @@ func validateFloatingIPs(p *openstack.Platform, ci *CloudInfo, fldPath *field.Pa
 	}
 	return allErrs
 }
+
+// validateExternalNetwork validates the user's input for the clusterOSImage and returns a list of all validation errors
+func validateClusterOSImage(p *openstack.Platform, ci *CloudInfo, fldPath *field.Path) (allErrs field.ErrorList) {
+	if p.ClusterOSImage == "" {
+		return
+	}
+
+	// If the user input is an http address, then skip the validation
+	if _, err := url.ParseRequestURI(p.ClusterOSImage); err == nil {
```

Comment:
this comment does not match what is being done here, it's checking for a valid URI which includes any scheme not just http.

if you support http and https only, add validation for that here.